tests/ffi/Cargo.toml

```diff
@@ -9,7 +9,7 @@ publish = false
 path = "lib.rs"
 
 [dependencies]
-cxx = { path = "../..", default-features = false }
+cxx = { path = "../..", default-features = true }
```

Comment:
will revert though I'm not entirely sure what the point of this Cargo.toml is. It fails to build without this change (I used this just to run cargo expand on the tests more easily while debugging)